settings.gradle

```diff
@@ -9,24 +9,29 @@ includeWithFlags ':bom', 'bom'
 includeWithFlags ':client:java',                            'java', 'publish', 'relocate'
 includeWithFlags ':client:java-armeria',                    'java', 'publish', 'relocate'
 includeWithFlags ':client:java-armeria-legacy',             'java', 'publish', 'relocate'
-includeWithFlags ':client:java-spring-boot2-autoconfigure',  'java', 'publish', 'relocate', 'no_aggregation'
-includeWithFlags ':client:java-spring-boot2-starter',        'java', 'publish', 'relocate', 'no_aggregation'
+includeWithFlags ':client:java-spring-boot2-autoconfigure', 'java', 'publish', 'relocate', 'no_aggregation'
+includeWithFlags ':client:java-spring-boot2-starter',       'java', 'publish', 'relocate', 'no_aggregation'
 includeWithFlags ':client:java-spring-boot3-autoconfigure', 'java17', 'publish', 'relocate'
 includeWithFlags ':client:java-spring-boot3-starter',       'java17', 'publish', 'relocate'
 includeWithFlags ':common',                                 'java', 'publish', 'shade', 'trim'
 includeWithFlags ':common-legacy',                          'java', 'publish', 'relocate'
+includeWithFlags ':mirror',                                 'java17', 'publish', 'relocate'
```

Comment:
How about renaming to `:server-mirror-git` and making it focus on `GitMirror` implementation only?